__tests__/app.test.js

```diff
@@ -31,6 +35,18 @@ describe("GET /api/incorrect-path", () => {
       .get("/api/wrong-path")
       .then((response) => {
         expect(response.status).toBe(404);
+        expect(response.error.message).toBe("cannot GET /api/wrong-path (404)");
+      });
+  });
+});
+
+describe("GET /api", () => {
+  test("returns 200 status code and endpoint documentation", () => {
+    return request(app)
+      .get("/api")
+      .then((response) => {
+        expect(response.status).toBe(200);
```

Comment:
I have noticed that you have not made use of supertest's built in .expect method which takes the statuscode you expect to get back! 

The way you have done it is perfectly fine, but just a heads up that there is an easier way to do this 👍 !